# Inspecting a DataFrame

#### EXERCISE:
We've loaded the credit card records of our four suspects into a DataFrame called <code>credit_records</code>.  Let's learn more about the structure of this DataFrame.

The <code>pandas</code> module has been imported under the alias <code>pd</code>.
The DataFrame <code>credit_records</code> has already been imported.

<hr>

How many rows are in <code>credit_records</code>?

#### INSTRUCTION:
* Use the <code>.info()</code> method to inspect the DataFrame <code>credit_records</code>

#### SCRIPT.PY:

In [1]:
import pandas as pd
credit_records = pd.read_csv("credit_records.csv")
#Use .info() to inspect the DataFrame credit_records
print(credit_records.info())

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 104 entries, 0 to 103
Data columns (total 5 columns):
suspect     104 non-null object
location    104 non-null object
date        104 non-null object
item        104 non-null object
price       104 non-null float64
dtypes: float64(1), object(4)
memory usage: 4.1+ KB
None


#### QUESTION:
How many rows are in <code>credit_records</code>?

#### POSSIBLE ANSWERS:
* 103
* 104
* 5
* 64